v2/support/requirements_sample_app/assets/iframe.html

```diff
@@ -0,0 +1,48 @@
+<html>
+<head>
+  <link rel="stylesheet" href="https://assets.zendesk.com/apps/sdk-assets/css/1/zendesk_garden.css" type="text/css">
+</head>
+<body>
+  <section data-main>Loading...</section>
+  <!-- https://github.com/zendesk/zendesk_app_framework_sdk -->
+  <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
+  <script>
+    // Initialise the Zendesk JavaScript API client
+    // https://developer.zendesk.com/apps/docs/apps-v2
+    var client = ZAFClient.init();
+
+    function renderText(text) {
+      var mainSectionEl = document.querySelector('section[data-main]');
+      mainSectionEl.innerText = text;
+    }
+
+    function init() {
+      client.get('requirement:ticket_custom_field').then(function(data) {
+        var requirement = data['requirement:ticket_custom_field'];
+
+        if (!requirement) {
+          renderText('Error: Could not find `requirement:ticket_custom_field`, have you added the field to your ticket form?');
+          return;
+        }
+
+        var customFieldPath = 'custom_field_' + requirement.requirement_id;
+        // set the text to the current value
+        client.get('ticket.customField:' + customFieldPath).then(function(data) {
```

Comment:
I can replace the duplicate strings for variables, but would add an extra line for each var definition. Didn't seem like much of an improvement.